metricbeat/module/elasticsearch/index_summary/data.go

```diff
@@ -31,52 +31,56 @@ import (
 
 var (
 	schema = s.Schema{
-		"primaries": c.Dict("primaries", s.Schema{
-			"docs": c.Dict("docs", s.Schema{
-				"count":   c.Int("count"),
-				"deleted": c.Int("deleted"),
-			}),
-			"store": c.Dict("store", s.Schema{
-				"size": s.Object{
-					"bytes": c.Int("size_in_bytes"),
-				},
-			}),
-			"segments": c.Dict("segments", s.Schema{
-				"count": c.Int("count"),
-				"memory": s.Object{
-					"bytes": c.Int("memory_in_bytes"),
-				},
-			}),
-		}),
-		"total": c.Dict("total", s.Schema{
-			"docs": c.Dict("docs", s.Schema{
-				"count":   c.Int("count"),
-				"deleted": c.Int("deleted"),
-			}),
-			"store": c.Dict("store", s.Schema{
-				"size": s.Object{
-					"bytes": c.Int("size_in_bytes"),
-				},
-			}),
-			"segments": c.Dict("segments", s.Schema{
-				"count": c.Int("count"),
-				"memory": s.Object{
-					"bytes": c.Int("memory_in_bytes"),
-				},
-			}),
-		}),
+		"primaries": c.Dict("primaries", indexSummaryDict),
+		"total":     c.Dict("total", indexSummaryDict),
 	}
 )
 
-func eventMapping(r mb.ReporterV2, info elasticsearch.Info, content []byte) error {
-	var event mb.Event
-	event.RootFields = common.MapStr{}
-	event.RootFields.Put("service.name", elasticsearch.ModuleName)
+var indexSummaryDict = s.Schema{
+	"docs": c.Dict("docs", s.Schema{
+		"count":   c.Int("count"),
+		"deleted": c.Int("deleted"),
+	}),
+	"store": c.Dict("store", s.Schema{
+		"size": s.Object{
+			"bytes": c.Int("size_in_bytes"),
+		},
+	}),
+	"segments": c.Dict("segments", s.Schema{
+		"count": c.Int("count"),
+		"memory": s.Object{
+			"bytes": c.Int("memory_in_bytes"),
+		},
+	}),
+	"indexing": indexingDict,
+	// following field is not included in the Stack Monitoring UI mapping
+	"bulk":   elasticsearch.BulkStatsDictMetricbeatCompatible,
```

Comment:
Any reason `BulkStatsDictMetricbeatCompatible` is in the `elasticsearch` package and not local to this package here and unexported (like `searchDict`, `indexingDict`, etc.)?

Reply:
I just wanted to maintain coherence with https://github.com/elastic/beats/blob/master/metricbeat/module/elasticsearch/elasticsearch.go#L116
Now, after you mention this, I have realized that it's not used anywhere but in the `index_summary` so we can actually remove both from the `elasticsearch.go` file.